### ISSUE TITLE
Opening  images fails on Fedora 35 and Centos 7

### ISSUE THREAD

**Issue by zeratoss** [TissUUmaps/TissUUmaps#7]
Hi, I was able to use TissUUmaps on Windows and MacOS, however on my two linux systems the program just stays blank after opening an image file. The program was installed in a fresh conda enviroment according to the instructions.
Debug output below:

```
❯ tissuumaps -d
INFO:root: * TissUUmaps version: 3.0.9.5

DEBUG:root:Debug mode
INFO:root:Starting port detection
INFO:root:Ending port detection 5432
 * Serving Flask app 'tissuumaps' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
/home/wintermute/mambaforge/envs/tissuumaps_env/lib/python3.9/site-packages/werkzeug/serving.py:716: ResourceWarning: unclosed <socket.socket fd=70, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('0.0.0.0', 0)>
  self.socket = socket.fromfd(fd, address_family, socket.SOCK_STREAM)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
INFO:werkzeug: * Running on http://127.0.0.1:5432 (Press CTRL+C to quit)
DEBUG:root:Path from referrer: /mnt/data/shared
INFO:werkzeug:127.0.0.1 - - [14/Jun/2022 16:16:14] "GET / HTTP/1.1" 200 -
DEBUG:root:Path from referrer: /mnt/data/shared
INFO:werkzeug:127.0.0.1 - - [14/Jun/2022 16:16:15] "GET / HTTP/1.1" 200 -
DEBUG:root:Opening image: / http://127.0.0.1:5432/home/wintermute/Downloads/EASI_CK/CK_Base_1_stitched-1.tif PyQt5.QtCore.QUrl('http://127.0.0.1:5432/home/wintermute/Downloads/EASI_CK/CK_Base_1_stitched-1.tif')
INFO:werkzeug:127.0.0.1 - - [14/Jun/2022 16:16:26] "GET /CK_Base_1_stitched-1.tif?path=home/wintermute/Downloads/EASI_CK HTTP/1.1" 200 -


```
Any Ideas?

**Comment by cavenel** [comment]
Reading about Qtwebengine issues, I found this one: https://github.com/vlaci/openconnect-sso/issues/69
Could you test the workaround proposed there:
> Try to type, before starting TissUUmaps:
> 
> ```
> $ export QTWEBENGINE_CHROMIUM_FLAGS="--no-sandbox"
> ```

**Comment by zeratoss** [comment]
Hi cavenel, thank you for your message.

Exporting that variable did not make any differences.

However, the browser link starts the image conversion process and the program behaves as expected.

**Comment by cavenel** [comment]
Hi @zeratoss,

Thanks for the feedback. If everything loads fine in the browser, it is definitely a problem with the Qt5 gui. It is difficult to debug without reproducing it.
I will see if I can test on a Fedora 35 or CentOS 7 virtual machine, and come back to you.

For now, you can use the TissUUmaps server instead, via the command line `$ tissuumaps_server /path/to/image/folder/ -p 5001`, and open the image in the browser directly on http://127.0.01:5001.

**Comment by cavenel** [comment]
Hi,

Could you test reinstalling TissUUmaps with pip directly from the last version on GitHub?
```bash
pip install https://github.com/TissUUmaps/TissUUmaps/archive/master.zip
```
It should add the "--no-sandbox" automatically, and fix a small bug that was blocking the flag from working.
Tested on Fedora 35 but not Centos 7.

**Comment by zeratoss** [comment]
I can confirm it works now on Fedora.

Still blank on Centos

**Comment by cavenel** [comment]
When it is blank, can you try to click on "File > Close image" in the TissUUmaps menu?
I was not able to reproduce the blank screen in CentOS 7.

**Comment by cavenel** [comment]
TissUUmaps 3.1 is now using PyQt6 instead of PyQt5, so this issue is not relevant anymore. Please open a new issue if you still have issues in Linux.
